src/app/components/lists/list-overview-item/list-overview-item.component.css

```diff
@@ -8,4 +8,11 @@ li{
 #listSize{
     max-height: 118px;
     overflow:hidden;
-}
\ No newline at end of file
+}
+.deleteListBtn{

```

Comment:
Dont call it btn if its only attached to an icon. Also if its designed for 1 element do id instead of class